src/neural_net_output.jl

```diff
@@ -26,7 +26,7 @@ n_dist_parameters(::Type{D}) where D = length(parameter_functions(D))
 "ensures the parameter is a legal value"
 function parameter_functions end
 parameter_functions(::Type{<:Normal}) = (identity, abs)
-parameter_functions(::Type{<:LogNormal}) = (identity, abs∘log∘abs)
+parameter_functions(::Type{<:LogNormal}) = (identity, abs)
```

Comment:
This function is kinda weird to use, vaguely motivated by the sensitivity of the output to this parameter, but really just was there as a hack.

Once we clip the gradients we do not need it anymore